examples/timer/index.ts

```diff
@@ -0,0 +1,11 @@
+import * as azure from '@pulumi/azure';
+
+const resourceGroup = new azure.core.ResourceGroup('example', { location: 'West US 2' });
+
+new azure.appservice.TimerSubscription('everyminute', {
+    resourceGroup,
+    schedule: { second: 0 },
```

Comment:
i'm not certain how `second: 0` translates to `everyminute`.

Reply:
It translated to "0 * * * * *" which means "every time second = 0" means every minute. Not super obvious, like most cron expressions...

Reply:
for the purposes of an example how about `{ minute: 5 }` and call it `everyfiveminutes` :)

Reply:
`{ minute: 5}` will trigger when minute is 5, every second. So, 60 times per hour. Every five minutes is `0 */5 * * * *`.

Reply:
ok.  can we do `{ second: 0, minute: 5 }` :D

Reply:
That is going to run once per hour, at X:05:00